test/fixtures/entryFiles/deps/default.mjs

```diff
@@ -0,0 +1 @@
+import internalDep from 'internalDep';
```

Comment:
As internal imports must be relative (with extension) in ESM, i'm not sure to understand here.
Should it be `import externalDep from 'externalDep';` ?
Should it be `import internalDep from './internalDep.js';` + need to create `internalDep.js` file ?
If i'm not wrong, there are no ESM-import tests, should we add some ?

Same question in `default.js` -> `require('internalDep')` what is intended ?